conversion_rules.py

```diff
@@ -172,8 +172,10 @@
 	'Coalition.rte/Devices/Sprites/CannonRound.bmp': 'Imperatus.rte/Devices/Weapons/Devastator/DevastatorRound.bmp',
 	'Coalition.rte/Devices/Sprites/RocketB.bmp': 'Coalition.rte/Devices/Weapons/MissileLauncher/MissileB.bmp',
 	'Coalition.rte/Devices/Sprites/RocketA.bmp': 'Coalition.rte/Devices/Weapons/MissileLauncher/MissileA.bmp',
-    'Coalition.rte/Devices/Sprites/UberShotCluster.bmp' : 'Coalition.rte/Devices/Weapons/UberCannon/UberShotCluster.bmp',
-	'Coalition.rte/Devices/Sprites/MuzzleAutoCannonA.bmp' : 'Coalition.rte/Effects/Pyro/Flashes/MuzzleAutoCannonA.bmp',
+	'Coalition.rte/Devices/Sprites/UberShotCluster.bmp' : 'Coalition.rte/Devices/Weapons/UberCannon/UberShotCluster.bmp',
+	'Coalition.rte/Devices/Sprites/UberShot.bmp' : 'Coalition.rte/Devices/Weapons/UberCannon/UberShot.bmp',
+	'Coalition.rte/Devices/Sprites/MagazineCannonUberA.bmp' : 'Coalition.rte/Devices/Weapons/UberCannon/UberCannonMagazine.bmp',
+  'Coalition.rte/Devices/Sprites/MuzzleAutoCannonA.bmp' : 'Coalition.rte/Effects/Pyro/Flashes/MuzzleAutoCannonA.bmp',
```

Comment:
Can you confirm that I've merged these correctly? I'm relatively new to Github and all, i just know that the original changed  before I made this pull request, so I believe I've fixed the conflicts

Reply:
It seems good, only a single line got added, like you wanted.